### ISSUE TITLE
'Start containers' button should be visible even so there is an update available

### ISSUE THREAD

**Issue by Selaron** [nextcloud/all-in-one#4618]
### Is your feature request related to a problem? Please describe. 

When there is an AIO upgrade available and the containers are stopped for whatever reason, the AIO dashboard does not seem to allow container start without upgrade, the only button visible reads like "Upgrade nextcloud".

### Describe the solution you'd like

The button "Start containers" should be visible.

### Additional context

I had a problem performing the upgrade via AIO dashboard and would have liked to just start the (outdated) nextcloud containers in order to bring back the service while I'd have the time to do my research on the actual problem.

FYI The root cause of my problem seems to be a misconfigured watchtower that is unable to perform docker actions as I am running docker rootless.
I helped my self by upgrading the AIO master container via CLI: `docker compose pull && docker compose up -d`

This is the log:

```
time="2024-05-03T16:34:30Z" level=debug msg="Sleeping for a second to ensure the docker api client has been properly initialized."
time="2024-05-03T16:34:31Z" level=debug msg="Making sure everything is sane before starting"
time="2024-05-03T16:34:31Z" level=info msg="Watchtower 1.7.1"
time="2024-05-03T16:34:31Z" level=info msg="Using no notifications"
time="2024-05-03T16:34:31Z" level=info msg="Only checking containers which name matches \"nextcloud-aio-mastercontainer\""
time="2024-05-03T16:34:31Z" level=info msg="Running a one time update."
time="2024-05-03T16:34:31Z" level=debug msg="Checking containers for updated images"
time="2024-05-03T16:34:31Z" level=debug msg="Retrieving running containers"
time="2024-05-03T16:34:31Z" level=error msg="Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x9fa4e2]

goroutine 1 [running]:
github.com/containrrr/watchtower/pkg/metrics.NewMetric({0x0, 0x0})
	/home/runner/work/watchtower/watchtower/pkg/metrics/metrics.go:31 +0x22
github.com/containrrr/watchtower/cmd.runUpdatesWithNotifications(0xc000205950)
	/home/runner/work/watchtower/watchtower/cmd/root.go:377 +0x171
github.com/containrrr/watchtower/cmd.Run(0xc000234300?, {0xc000093a40?, 0x4?, 0x4?})
	/home/runner/work/watchtower/watchtower/cmd/root.go:168 +0x658
github.com/spf13/cobra.(*Command).execute(0xc000234300, {0xc0000300b0, 0x4, 0x4})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xaa7
github.com/spf13/cobra.(*Command).ExecuteC(0xc000234300)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x417
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/containrrr/watchtower/cmd.Execute()
	/home/runner/work/watchtower/watchtower/cmd/root.go:75 +0x52
main.main()
	/home/runner/work/watchtower/watchtower/main.go:13 +0x17
```

The `docker.sock` is accessible to nextcloud-aio-mastercontainer via volume mount `/run/user/1230/docker.sock:/var/run/docker.sock:ro`.
I guess AIO just does not "know" about that fact and tells watchtower to use  `/var/run/docker.sock`. Did I miss out a point on your really great(!) step-by-step docker rootless AIO install howto?

**Comment by szaimen** [comment]
Hi, see https://github.com/containrrr/watchtower/issues/1942

**Comment by Selaron** [comment]
"closed as completed" so did you implement my requested enhancement?
This is not a duplicate of the linked issue.
